file_organizer_v2/src/file_organizer/cli/profile.py

```
@@ -0,0 +1,509 @@
+            click.echo(f"  Updated: {profile.updated}")
+
+            # Show statistics
+            global_prefs = len(profile.preferences.get('global', {}))
+            dir_prefs = len(profile.preferences.get('directory_specific', {}))
+            click.echo(f"  Preferences: {global_prefs} global, {dir_prefs} directory-specific")
+
+        click.echo("\n" + "=" * 80)
+
+    except Exception as e:
+        click.echo(f"Error listing profiles: {e}", err=True)
+        raise click.Abort()
```

Ruff (0.14.13)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)